### PR TITLE
Adding note about Tenant to consider when RHM is enabled.

### PR DESCRIPTION
**Pull request by dilip-2201**
Adding below note:
Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be disabled in all regions. You must execute this cmdlet in each region that are added in Allowed Data Location for Skype for Business.

### CODE REVIEW

File: skype/skype-ps/skype/Disable-CsOnlineSipDomain.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ This cmdlet should not be run for domains that contain users hosted in Skype for
 - These users will no longer be able to login to the Skype for Business Online environment.
 - If these users use Teams, they will no longer be able to inter-operate with Skype for Business users, nor will they be able to federate with any users in other organizations.
 
+Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be disabled in all regions. You must execute this cmdlet in each region that are added in Allowed Data Location.
```
Comment (dariomws) [inline skype/skype-ps/skype/Disable-CsOnlineSipDomain.md:49]:
```suggestion
Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be disabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location.  
```

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Disable-CsOnlineSipDomain.md (hunk 1 above)
Comment (dariomws) [reply skype/skype-ps/skype/Disable-CsOnlineSipDomain.md:49]: Likelihood here it should be "if disabled, sip domains must be disabled in all regions".

**Review by dariomws — APPROVED** [review]
@dilip-2201 my apologies, now with your comments I understand. And the note you added is practically like the warning (without specific regions). I approve it again just in case to avoid confusions.

Thank you so much for your explanation!

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne

**Comment by dilip-2201** [comment]
@get-itips @dariomws - Thanks for assisting with the change, however the first line should be called out "if the Tenant is enabled for Regionally Hosted Meetings" in both the cmdlets. (Not Disabled)

_If the Tenant is  **disabled** for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be disabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location._

**Comment by get-itips** [comment]
> @get-itips @dariomws - Thanks for assisting with the change, however the first line should be called out "if the Tenant is enabled for Regionally Hosted Meetings" in both the cmdlets. (Not Disabled)
> 
> _If the Tenant is **disabled** for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be disabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location._

So an edit is needed? can you create another PR? thanks

**Comment by dariomws** [comment]
@dilip-2201 fixed. I think after several comments and our discussion went wrong. Thanks for noticing!

**Pull request merged**